src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -81,7 +81,8 @@ const TopicFragment = ({
   setUnfollowedTopic,
   fetchData = () => {},
   searchText,
-  withoutRecent = false
+  withoutRecent = false,
+  initialTopics
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `initialTopics` prop is a good approach to improve the initial load time. However, it's not clear from this code snippet where `initialTopics` is coming from and how it's being updated. Make sure that `initialTopics` is properly initialized and updated in the parent component to ensure that the most recent topics are displayed when the component is first opened.
```diff
   withoutRecent = false,
+  initialTopics
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->